soc/arm/atmel_sam0/same51/soc.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2019 ML!PA Consulting GmbH
+ * Copyright (c) 2023 Sebastian Schlupp
```

Comment:
Normally, adding a few lines to an existing file does not warrant a copyright notice. Otherwise our files would have _really_ many copyright lines.